cmd/policy-assistant/pkg/cli/analyze.go

```diff
@@ -96,6 +104,10 @@ func SetupAnalyzeCommand() *cobra.Command {
 	command.Flags().StringVar(&args.TrafficPath, "traffic-path", "", "path to json traffic file, containing of a list of traffic objects")
 	command.Flags().StringVar(&args.ProbePath, "probe-path", "", "path to json model file for synthetic probe")
 	command.Flags().DurationVar(&args.Timeout, "kube-client-timeout", DefaultTimeout, "kube client timeout")
+	command.Flags().StringVar(&args.SourceWorkloadTraffic, "source-workload-traffic", "", "Source workload traffic in this form namespace/workloadType/workloadName")
```

Comment:
I would recommend renaming these two lines to `--source-workload` and `--destination-workload` for less typing?

Reply:
or even `--src-workload`, `dst-workload`?

Reply:
good idea!